test/lists.js

```diff
@@ -1,72 +1,272 @@
-const chai = require('chai')
-const expect = chai.expect
-
+const { expect } = require('chai')
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: 'demo' })
+const fakeHubspotApi = require('./helpers/fake_hubspot_api')
+const { createTestContact, deleteTestContact } = require('./helpers/factories')
+
+describe('lists', function() {
+  const hubspot = new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'some-fake-token',
+  })
+  const listProperties = { name: 'Test list name' }
```

Comment:
Thoughts on updating this (and similar constants) to include the word "contact"? So this would become `contactListProperties`?

Reply:
I think I'm going to leave this as is to keep the test vernacular the same as what we have in the code.